doc/concepts/data-concepts/repo.md

```diff
@@ -0,0 +1,33 @@
+# Repository
+
+A Pachyderm repository is a location where you store your data inside
+Pachyderm. A Pachyderm repository is a top-level data object that contains
+files and folders. Similar to Git, a Pachyderm repository tracks all
+changes to the data and creates a history of data modifications that you
+can access and review. You can store any type of file in a Pachyderm repo,
+including binary and plain text files.
+
+Unlike a Git repository that stores history in a `.git` file in your copy
+of a Git repo, Pachyderm stores the history of your commits in a centralized
+location in the etcd key-value store. Because of that, you do not run into
+merge conflicts as you often do with Git commits when you try to merge
+your `.git` history with the master copy of the repo. With large datatsets
+resolving a merge conflict might not be possible.
+
+A Pachyderm repository is the first entity that you configure to create
+a standard pipeline. You can create a repository by running the
+`pachctl create repo <name>` command or by using the Pachyderm UI. After
+creating the repository, you can add your data by using the
+`pachctl put file <repo@branch:/path><path-to-filesystem>` command. The
+path to the filesystem can be a local directory, file, or a URL.
+This repository is called the *input repo* and is stored under
+`pfs/<repo-name>`. All the input data for your pipeline is stored in
+this repository.
+
+After you create a repository, you can specify that repository as a
```

Comment:
> After you create a repository, you can specify that repository as a
PFS input in your pipeline. When the pipeline detects
new, unprocessed data in the repository, it runs your code against it.

remove this, it'll be covered in Pipelines. No need to bloat the sections. If you want, you can change this to something like, "Both outside applications and Pachyderm pipelines can read and process data from repos".

> For each pipeline, Pachyderm automatically creates a `pfs/out` directory
in the `pachd` container. This directory is called the *output repository*.
All your output data is stored in this directory.

This is incorrect. As above, we should just remove this section, it belongs in pipelines, but I'll explain what's wrong and we should discuss in TOH. This stuff is tricky -- it's not just about picking up on the names of things (e.g. pfs/out, pachd, etc), but actually building an understanding of the concepts and how they interact. That's exactly what I'm here for and would love to go over this with you as many times as you want. :D

- pfs/out is not created in the pachd container, it's created in the user container
- the above is not done for each pipeline, it's actually done for each datum within a job (to be discussed in Jobs/datums/pipelines)
- The directory is not called the output repository. The output repository is called the output repository :P. That directory is pfs/out, which is where the user code should write data that it want put into the output repository. 
- Output data is not stored in the pfs/out directory. In fact, the pfs/out directory is wiped clean after every datum. The output data is stored in the output repo.